distmv/normal.go

```diff
@@ -246,6 +246,29 @@ func (n *Normal) Prob(x []float64) float64 {
 	return math.Exp(n.LogProb(x))
 }
 
+// Quantile returns the multi-dimensional inverse cumulative distribution function.
+// len(x) must equal len(p), and if x is non-nil, len(x) must also equal len(p).
```

Comment:
I don't understand this.

Also, at the moment there is no explicit length check for `x` when `x != nil`; the panic is potentially a runtime bounds check if `len(x) < len(p)`, otherwise a panic in `TransformNormal`.